crates/sncast/tests/e2e/account/deploy.rs

```diff
@@ -38,11 +40,10 @@ pub async fn test_happy_case(class_hash: &str, account_type: &str) {
         URL,
         "--name",
         "my_account",
-        "--max-gas",
-        "1000",
-        "--max-gas-unit-price",
-        "100000000000",
-    ];
+    ]
+    .into_iter()
+    .chain(TEST_RESOURCE_BOUNDS_FLAGS.into_iter())
+    .collect::<Vec<&str>>();
```

Comment:
Nit, but maybe this could be hidden behind some helper function?

Reply:
Sure, this should be better:

```rust
let args = vec![
        "--accounts-file",
        "accounts.json",
        "--account",
        "my_account",
        "deploy",
        "--url",
        URL,
        "--class-hash",
        MAP_CONTRACT_CLASS_HASH_SEPOLIA,
        "--salt",
        "0x2",
        "--unique",
    ];
    let args = apply_test_resource_bounds_flags(args);
```